src/library-authoring/create-library/index.scss

```diff
@@ -1,39 +1,233 @@
-.pgn__alert-modal {
-    max-width: 450px;
+.form-create-alert {
+    margin-top: ($baseline*1.6);
+}
+
+.dropdown-container {
+    @include pgn-box-shadow(1, 'centered');
 
-    .pgn__modal-title {
-        @include font-size(22);
-        @include line-height(28);
+    border-radius: ($baseline*0.2);
+    max-height: 200px;
+    font-size: 1rem;
+    font-weight: normal;
+    line-height: 1.25rem;
+    overflow-y: scroll;
+    position: absolute;
+    background-color: $white;
+    width: 100%;
+    z-index: 100 !important;
 
-        font-weight: 700;
+    .dropdown-item {
+        font-size: 1rem;
+        line-height: 1.25rem;
+        padding: ($baseline*0.5) ($baseline*0.8);
     }
+}
+
+.dropdown-group-wrapper {
+    position: relative;
+}
+
+$baseline: 20px;
```

Comment:
Isn't this variable in the file `studio.scss`?
is it needed here additionally?

Reply:
I added it because we use it in this file and it's better than import all `studio.scss` file